mk/jsonnet.mk

```diff
@@ -0,0 +1,17 @@
+JSONNET_DIR=$(KUMA_DIR)/app/kumactl/data/install/k8s/metrics/grafana/jsonnet
+JSONNET_PKG_HOME=../../../../../../../../build/jsonnet
```

Comment:
It would be nice to get rid of the relative path here. One way to do it is to remove `cd $(JSONNET_DIR)` at the beginning of each target, so each target is executed from the project's root. I guess this will require moving `jsonnetfile.json` and `jsonnetfile.lock.json` to the project's root, but I'm fine with that.